mne/viz/backends/_pysurfer_mayavi.py

```diff
@@ -58,7 +58,7 @@ class _Renderer(_BaseRenderer):
     """
 
     def __init__(self, fig=None, size=(600, 600), bgcolor=(0., 0., 0.),
-                 name=None, show=False):
+                 name=None, show=False, **kwargs):
```

Comment:
I don't much like `kwargs` because it's easy to "lose" arguments. Why not just `smooth_shading=True` here?

Reply:
I wanted to use this property of `kwargs` to introduce backend specific parameter actually, you don't like the idea? 
Of course, I can just put it to a plain parameter indeed.

Reply:
what is your opinion on this @agramfort ?